docs/AddACalloutBox.md

```diff
@@ -0,0 +1,21 @@
+# Add a call out box
+
+Adding a call out box to a Markdown document is easy. In most cases, you should use Markdown definition list syntax.
+
+	Are you eligible for disability benefits?
+	: Yes if:
+
+    	-	You participated in Projects 112 or SHAD testing.
+    	
+	Who is covered
+	: You and your survivors
+	{:.va-callout}
+
+Use the syntax found in [Adding Attributes](AddingAttributes.md) to add the `.va-callout` class to the end of the call out list.
+
+The syntax here is very specfic.
+
+- Answers / definitions should be left-aligned with the question or definition term and begin with `:`.
```

Comment:
These are a little confusing to me. When we say `There should be a single line break between the question or term and the answer`, do we mean between `: Yes if:` and `    -   You participated in Projects 112 or SHAD testing.` or between `Are you eligible for disability benefits?` and `: Yes if:`. This is only confusing to me because it looks like the line break if after `: Yes if:`, however, `Are you eligible for disability benefits?` is the "question".

Maybe it is helpful to label the parts of this section like:
```
Are you eligible for disability benefits?     // <---- Definition
: Yes if:                                                     // <---- Term

    -   You participated in Projects 112 or SHAD testing.          // <---- Answer
```

Also, I doubt the example above is correct.